port/raspberrypi/rp2xxx/src/hal/irq.zig

```diff
@@ -139,3 +169,48 @@ pub fn disable(mask: Mask) void {
         },
     }
 }
+
+const VtorFunction = *const fn () callconv(.C) void;
```

Comment:
nit: I guess it still works, but zig moved these sort of enums to lower case
```suggestion
const VtorFunction = *const fn () callconv(.c) void;
```